### PR TITLE
Adding ci to osd-network-verifier

### PR DESCRIPTION
**Pull request by bng0y**

### CODE REVIEW

File: ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    tag: golang-1.16
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: test
```
Comment (boranx) [inline ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml:15]: we may consider adding also `make build` as a condition as it has fmt&mod checks

File: ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+resources:
+  '*':
+    limits:
+      memory: 4Gi
```
Comment (boranx) [inline ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml:10]: (optional)isn't it a bit much for a build?

**Review by boranx — COMMENTED** [review]
thanks for adding the pipeline support! here are my 2 cents as review

File: ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml (hunk 2 above)
Comment (bng0y) [reply ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml]: turns out this limit is actually necessary
```
  * could not run steps: step build failed: test "build" failed: the pod ci-op-5gi2xhwr/build failed after 40s (failed containers: test): ContainerFailed one or more containers exited
Container test exited with code 2, reason OOMKilled
```

**Pull request merged**